packages/telemetry/src/flight-recorder.js

```diff
@@ -387,6 +387,6 @@ export const makeSlogSenderFromBuffer = ({ writeCircBuf }) => {
  * @type {import('./index.js').MakeSlogSender}
  */
 export const makeSlogSender = async opts => {
-  const { writeCircBuf } = await makeMemoryMappedCircularBuffer(opts);
+  const { writeCircBuf } = await makeSimpleCircularBuffer(opts);
```

Comment:
Should it be an option for now if we're keeping both?

Reply:
I assumed I was the only one interested in keeping both so once I had confidence in the new one I removed the old one.

Let me know if you think the mmap one should stick around until a later day

Reply:
I'm fine removing. When I first started to review, it seemed it was sticking around.